run.sh

```diff
@@ -325,8 +325,8 @@ done
 # ======================================= DEFAULTS ============================================
 [[ -z "${PLATFORM}" ]] && PLATFORM=local
 [[ -z "${BINDER}" ]] && BINDER=rabbit
-[[ -z "${STREAM_APPS_VERSION}" ]] && STREAM_APPS_VERSION=Celsius.SR3
-[[ -z "${TASKS_VERSION}" ]] && TASKS_VERSION=Elston.RELEASE
+[[ -z "${STREAM_APPS_VERSION}" ]] && STREAM_APPS_VERSION=Einstein-SR3
```

Comment:
I'd suggest that we _always_ default to "latest" links. It is better not to pin it to a released version. It's OK, we can always run the ATs on the latest.

![image](https://user-images.githubusercontent.com/1562654/59572392-c1d99380-9061-11e9-984b-58334c03217a.png)
![image](https://user-images.githubusercontent.com/1562654/59572400-caca6500-9061-11e9-8e93-0154597d2a18.png)



That was the scope of the parent issue (at least as far as the title goes; description, of course, is super dated)

Reply:
ok, the above information helps. Updated the PR to use the `latest` by default and still supporting configurable stream/task apps version.